### ISSUE TITLE
Specifying MSVC Toolset v141 Produces v140 SDK

### ISSUE THREAD

**Issue by geoffthemedio** [freeorion/freeorion-sdk#34]
I've installed MSVC 2017 and CMake 3.9.3. I then used the provided instructions to produce the SDK, additionally specifying that MSVC 2017 and toolset v141 should be used. This runs, and generates a .zip as expected.

However it appears to have still used the v140 (MSVC 2015) toolset. This can be seen from filenames and looking into binary dependencies; all the boost libraries and DLLs have `vc140-mt-1_59` in their filenames, and matching dependencies in Dependency Walker. `python27.dll` and `libpng16.dll` depend on `VCRUNTIME140.DLL` in Dependency Walker.

Output from generating the SDK:

[build_CMake_step_1.txt](https://github.com/freeorion/freeorion-sdk/files/1347110/build_CMake_step_1.txt)
[build_CMake_step_2.txt](https://github.com/freeorion/freeorion-sdk/files/1347111/build_CMake_step_2.txt)

**Comment by o01eg** [comment]
When I tried to use MSVC 2017 I have to bump boost version to latest available 1.63. Start at 1.64 boost should have support for 14.1:

https://github.com/freeorion/freeorion-sdk/compare/master...o01eg:master

**Comment by adrianbroher** [comment]
The latest freeorion-sdk master should support v141.  Can you check if it's working as expected?

@o01eg Is the diff you posted still relevant? Because it does some additional and strange things.

**Comment by o01eg** [comment]
@adrianbroher I also built x64 version and enabled verbosity to catch some errors so I'm not sure if entire diff is relevant.

**Comment by geoffthemedio** [comment]
It now seems to select the version 15 / 2017 compiler by default when running cmake:
````
C:\Users\Geoff\Desktop\freeorion-sdk\build>cmake ..
-- Building for: Visual Studio 15 2017
-- The C compiler identification is MSVC 19.11.25508.2
-- The CXX compiler identification is MSVC 19.11.25508.2
````

**Comment by adrianbroher** [comment]
The actual issue was already resolved before #41, so closing this now.
